### DISCUSSION TITLE
What are some good submods and how do I activate them?

### DISCUSSION THREAD

**Discussion by 09123731235** [Monika-After-Story/MonikaModDev#9990] [general - submod / external content]
Hi I'm pretty new to installing submods (tried it before and just got an error message) so I'm looking for any suggestions and a guide on how to install them. Apologies if that already exists I might just be dumb and not be able to find it ahaha. Thanks in advance for any responses!

**Comment by 09123731235** [comment]
is this a dumb question? i ask since its been 2 days with no responses.
i mostly just need help on how to use the submods as last time i tried it just gave an error message (but some suggestions on good submods would be helpful aswell).

**Comment by Wingdinggaster656** [comment]
No, this is not a dumb question. Sorry for being busy.

Take a look at this page: https://github.com/Monika-After-Story/MonikaModDev/wiki/External-Content

**Reply by 09123731235** [reply]
thanks for the reply and no prob was just worried it was a ridiculous question or smth

that page is good for finding mods but how do i use/activate? Also are there any there specifically you would recommend?

**Reply by 09123731235** [reply]
never mind i  have figured it out, sorry for any bother

**Reply by hiiiihelloo** [reply]
hi! how did you do it?
